tests/unit/param_types/test_guest_activity_notification_param.py

```diff
@@ -0,0 +1,253 @@
+import click
+import pytest
+
+from globus_cli.parsing.param_types.guest_activity_notify_param import (
+    GCSManagerGuestActivityNotificationParamType,
+    TransferGuestActivityNotificationParamType,
+)
+
+
+@click.command()
```

Comment:
The tests are failing for a relatively silly reason -- as of v8.2.0, `click` now strips suffixes of `_command` or `_cmd` from automatic command names.
You could do it some other way, but I think the simplest quick fix is this:

```suggestion
@click.command("activity-notifications-cmd")
```

Reply:
> The tests are failing for a relatively silly reason -- as of v8.2.0, `click` now strips suffixes of `_command` or `_cmd` from automatic command names. You could do it some other way, but I think the simplest quick fix is this:

I just now saw this because I was busy trying to figure out the cause for the test failure. The approach I took was to adopt a less stringent form of assert like the ones used for similar errors tests in `test_notification_param.py`.

https://github.com/globus/globus-cli/blob/af12bb90a6396f99f030b86a3c67f8c651fd4e6b/tests/unit/param_types/test_notification_param.py#L81-L93